mochify/lib/load-config.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const { cosmiconfig } = require('cosmiconfig');
+
+exports.loadConfig = loadConfig;
+
+async function loadConfig(config) {
+  const explorer = cosmiconfig('mochify', {
+    searchPlaces: [
+      'package.json',
+      '.mochifyrc',
+      '.mochifyrc.json',
+      '.mochifyrc.yaml',
+      '.mochifyrc.yml',
+      '.mochifyrc.js',
+      '.mochifyrc.cjs',
+      'mochify.config.js',
+      'mochify.config.cjs'
+    ]
+  });
+
+  const config_path = config.config;
+  const config_from_file = config_path
+    ? await explorer.load(resolveConfig(config_path))
+    : await explorer.search();
+
+  if (config_from_file) {
+    Object.assign(config, config_from_file.config, config);
```

Comment:
What's the reason for passing `config` twice? Would `Object.assign({}, config_from_file.config, config);` be easier to understand?

Reply:
`loadConfig` wasn't returning the config, it was "patching" the given object. I refactored this now.